css/style.css

```diff
@@ -0,0 +1,799 @@
+  font-family: "Shentox", Arial, Helvetica, sans-serif;
+  color: #aaa;
+  text-decoration: none;
+}
+
+.latest-news-headline h2 a {
+  text-decoration: none;
+  color: red;
+}
+
+/* .search {
+      display: block;
+      height: 48px;
+      padding: 16px;
+      width: 48px;
+    } */
+
+.search a:hover {
+  color: black;
+}
+
+/* .menu-icon {
+      display: block;
+      position: relative;
+      height: 16px;
+      padding: 16px;
+      width: 16px;
+    } */
+
+.spacer {
+  -webkit-box-flex: 1;
+  -ms-flex-positive: 1;
+  flex-grow: 1;
+}
+
+.main-nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  height: 48px;
+  border-bottom: 1px solid #eee;
+  list-style: none;
+  background-color: #fff;
+}
+
+.main-nav a:hover {
+  color: black;
+}
+
+/* .search,
+    .menu-icon { */
+.search {
+  border-left: solid 1px #eee;
+}
+
+.material-icons {
+  display: block;
+  width: 16px;
+  height: 16px;
+  margin-right: 10px;
+}
+
+.menu-icon span {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  margin-top: -1px;
+  left: 14px;
+  top: 50px;
+  background-color: #f42;
+}
+
+/* Cover Section */
+.cover {
+  width: 100%;
+  display: grid;
+  grid-template: repeat(2, 1fr) / 65% 35%;
+  padding: 4px;
+}
+
+.hidden-title {
+  display: none;
+  opacity: 0;
+}
+
+.article-big {
+  grid-area: 1 / 1 / 3 / 2;
+  margin: 4px;
+  background:
+    linear-gradient(
+      180deg,
+      rgba(2, 0, 36, 1) 0%,
+      rgba(9, 9, 121, 0) 0%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    linear-gradient(
+      320deg,
+      rgba(2, 0, 36, 1) 0%,
+      rgba(9, 9, 121, 0) 0%,
+      rgba(241, 62, 67, 1) 100%
+    ),
+    url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/10/Instagrammock-796x417.jpg);
+  background-size: cover;
+  height: 487.4px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  text-align: start;
+}
+
+.article-big-logo {
+  max-height: 96px;
+  max-width: 400px;
+  margin: 20px;
+}
+
+.article-small-top {
+  grid-area: 1 / 2 / 2 / 3;
+  margin: 4px;
+  background:
+    linear-gradient(
+      180deg,
+      rgba(2, 0, 36, 1) 0%,
+      rgba(9, 9, 121, 0) 0%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/12/1-13-796x417.jpg)
+      center;
+  background-size: cover;
+  height: 239.7px;
+}
+
+.article-small-bottom {
+  grid-area: 2 / 2 / 2 / 3;
+  margin: 4px;
+  background:
+    linear-gradient(
+      180deg,
+      rgba(2, 0, 36, 1) 0%,
+      rgba(9, 9, 121, 0) 0%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2018/03/hacker-sam-796x419.jpg)
+      center;
+  background-size: cover;
+  height: 239.7px;
+}
+
+.article-big .cover-article-info {
+  margin: 20px;
+}
+
+.article-big .article-section {
+  color: white;
+  font-size: 0.87 em;
```

Comment:
Line number 290 : invalid property value 
Kindly recommended to delete space (write 0.87em instead 0.87 em)